contentcuration/contentcuration/static/js/exercise_creation/models.js

```diff
@@ -0,0 +1,71 @@
+var Backbone = require("backbone");
+var _ = require("underscore");
+
+var ExerciseModel = Backbone.Model.extend({
+	defaults: {
+		title: "Exercise"
+	},
+
+	url: function() {
```

Comment:
FYI you can use [urlRoot](http://backbonejs.org/#Model-urlRoot) to the same effect here.